contents/blog/dev-marketing-for-startups.md

```diff
@@ -0,0 +1,98 @@
+
+You do not need a whole team of developers writing content, but have at least one to cover the more technical stuff (inc. tutorials). They’ll also help keep you connected to your users and not focused on purely top-of-funnel or general types of content. This has made a huge difference to the breadth of product content we’re able to cover. 
+
+**Be really careful about outsourcing** 
+
+You should outsource the stuff that you can’t do internally, not the stuff that you can do but don’t have capacity for. Otherwise you'll spend an inordinate amount of time managing a freelancer - when this goes wrong, it's harder work than just doing it yourself. 
+
+For example, we work with a freelancer on [deep dive technical topics](/blog/clickhouse-vs-postgres) that are interesting to our users but not strictly about PostHog itself. However, we’ve wasted a bunch of time hiring freelancers to write mediocre SEO content that we should have just done ourselves. 
+
+**Enforce code reviews on your content** 
+
+Every single piece of content at PostHog is edited by at least one other person. The writer has final say on what gets shipped, but it is everyone’s responsibility to give each other feedback. 
+
+Line edits are easy and less useful feedback to give. Instead, start big picture and then go into the detail. If the article isn’t genuinely useful or interesting, punctuation doesn’t matter. Feedback can include ‘ditch this article entirely, it’s not working.’
+
+**Hacker News is a double-edged sword** 
+
+We’ve found that hitting the front page results in a giant, ego-boosting traffic boost, with a noticeable but small signup boost. It’s not dependable though - even if you’re really good at it, you’ll have a 1 in 10 hit rate. You need to graduate to repeatable marketing at some point. 
+
+Don’t upvote your own content, and don’t ask other people to - post it and pray. There are no secret tricks.
+
+**Beware the attribution mirage**
+
+You can’t rely on UTM parameters to tell you where a user actually first heard about you. Example: user reads an article about PostHog on Hacker News -> searches ‘posthog’ -> clicks on a Google Ad. Our analytics will tell us ‘wow, Google Ads are awesome!’ But that’s not the whole picture. 
+
+In your product signup flow, include an optional free text box asking people where they first heard about you. About 10% of signups usually fill this in. Read the data and report on it every week. It’s manual but vitally important info.
+
+### Post-product-market fit
+
+**Paid ads are not all bad** 
+
+Paid ads can be a useful tool for learning quickly if you are building a consumer app, but _not_ for devtools (see ‘attribution mirage’ above), which is why I’ve put them in post-PMF. It is very easy to spend tons of money on shit, especially in Google Ads. NEVER USE THE DEFAULT SETTINGS on Google, LinkedIn, and Twitter especially. Hire a cheap agency. We aim to be 7/10 at paid ads, not amazing. Focus that energy on your content instead. 
```

Comment:
How much should you spend on a "cheap" agency?

Reply:
Added a note